frontend/src/metabase/visualizations/components/ChartSettingsWidgetPopover.tsx

```diff
@@ -38,12 +39,21 @@ const ChartSettingsWidgetPopover = ({
   const sectionNames = Object.keys(sections.current) || [];
   const hasMultipleSections = sectionNames.length > 1;
 
+  const onClose = () => {
+    const activeElement = document.activeElement as HTMLElement;
+    if (activeElement && contentRef.current?.contains(activeElement)) {
```

Comment:
Just curious if we need to check if the active element is inside the popover. Since it should be impossible to focus other input outside the popover while it's open (of course not programmatically, just by user interactions alone).

So, maybe we could do something like this, and don't need to have `contentRef` at all?
```ts
(document.activeElement as HTMLInputElement)?.blur();
```

What do you think?

Reply:
You can actually focus elements outside the popover if you tab to them (though I'm not entirely sure why anyone would do that). I think I'd prefer to keep the check just to limit the scope of what the component can impact.

Reply:
But I do get your point. I think in 99% of cases it would be totally fine

Reply:
Oh right. Totally forgot about the keyboard navigation 👍 Sounds good to keep the code as is for me.

Reply:
🙌